portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -52,20 +107,41 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
   }
 
   @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+  public void doGet(HttpServletRequest request, HttpServletResponse response)
+      throws IOException {
     response.setContentType("application/json");
 
-    Query query = new Query("Comment");
+    String numComments = request.getParameter("numComments");
+    String sortType = request.getParameter("sortType");
+    int nComments = 10;
```

Comment:
It's pretty confusion to have both "nComments" and "numComments". I'd suggest "numComments" and "numCommentsString".